api/src/controllers/utils.ts

```diff
@@ -55,9 +55,12 @@ router.post(
 			throw new InvalidPayloadError({ reason: `"hash" is required` });
 		}
 
-		const result = await argon2.verify(req.body.hash, req.body.string);
-
-		return res.json({ data: result });
+		try {
+			const result = await argon2.verify(req.body.hash, req.body.string);
+			return res.json({ data: result });
+		} catch (err: any) {
+			throw new InvalidPayloadError({ reason: err.message });
```

Comment:
The error returned for invalid input is
```json
{
  "errors": [
    {
      "message": "Invalid payload. pchstr must contain a $ as first char.",
      "extensions": {
        "reason": "pchstr must contain a $ as first char",
        "stack": "DirectusError: Invalid payload. pchstr must contain a $ as first char.\n    at <anonymous> (/Users/..../api/src/controllers/utils.ts:62:10)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)",
        "code": "INVALID_PAYLOAD"
      }
    }
  ]
}
```
As no `pchstr` was passed this is a confusing response. We should ideally provide a more generic `Verification Failed` or the like.